src/constants/gameplay.asm

```diff
@@ -131,17 +131,19 @@ DIALOG_GOT_ROD            equ $04
 DIALOG_GOT_GUARDIAN_ACORN equ $05
 
 ; Values for wLinkMotionState
-LINK_MOTION_INTERACTIVE    equ $00
-LINK_MOTION_SWIMMING       equ $01
-LINK_MOTION_JUMPING        equ $02
-LINK_MOTION_MAP_FADE_OUT   equ $03
-LINK_MOTION_MAP_FADE_IN    equ $04
-LINK_MOTION_REVOLVING_DOOR equ $05
-LINK_MOTION_FALLING_DOWN   equ $06
-LINK_MOTION_PASS_OUT       equ $07
-LINK_MOTION_RECOVER        equ $08
-LINK_MOTION_TELEPORT       equ $09
-LINK_MOTION_UNKNOWN        equ $0F
+; Only the first two values are interactive
+LINK_MOTION_DEFAULT              equ $00 ; ground, flying with rooster, etc
+LINK_MOTION_SWIMMING             equ $01
+LINK_MOTION_TYPE_NON_INTERACTIVE equ $02
+LINK_MOTION_UNSTUCKING           equ $02
```

Comment:
Still not entirely sure about this one.

It seems to be related to two cases:
- When Link transitions to a room on the south (bottom), it he lands on a colliding object, make it jump like a ledge until he finds some ground.
- Specifically, when on the Evil Eagle boss room and jumping from the top half to the bottom half of the tower, there's code checking if Link landed _inside_ the tower, and unstuck him if needed.

All of this seems very specific. Unstucking the player is good, but why doing it only when transitioning to the bottom?